content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -344,6 +344,27 @@ This ensures the packaged archive is as small as possible.
 
 But, if you are using `serverless-webpack`, you cannot use that method.
 
+### Lambdas with arm64 architectures
+
+Lambda functions that use [arm64 architectures (AWS Graviton2 processor)](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html#foundation-arch-adv) should use an `arm64` precompiled binary.
+
+Set in the Prisma schema the following in the generator block:
+
+```
+binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
```

Comment:
Is Lambda really still on OpenSSL 1.0.x?

Reply:
Hi @janpio, I've tried both `linux-arm64-openssl-1.0.x` and `linux-arm64-openssl-1.1.x` and it appears OpenSSL 1.1.x does not work with AWS Lambda. I've searched the AWS Lambda docs but couldn't find any information on whether it is still on `OpenSSL 1.0.x`. It might be worth opening a support case with AWS to follow it up.

I've made a minimal reproduction repo if you're like to try it out. It uses the AWS Lambda NodeJS 16.x runtime in `us-east-1` with the Serverless Framework.

- [OpenSSL 1.0.x branch](https://github.com/adamjq/aws-lambda-arm64-prisma)
- [OpenSSL 1.1.x branch](https://github.com/adamjq/aws-lambda-arm64-prisma/tree/arm64-openssl-1.1.x)

Reply:
Lambda runtimes are defined here https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html, mostly Amazon Linux 2
The [August update](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-20220823.html) shows 
```
openssl-1.0.2k-24.amzn2.0.4.aarch64
openssl-1.0.2k-24.amzn2.0.4.x86_64
```
So openssl 1.0.x